mod.json

```diff
@@ -37,7 +37,10 @@
                 "resources/deafen-icon-off.png",
                 "resources/deafen-icon-on.png"
             ]
-        }
+        },
+        "sprite": [
+            "resources/*.png"
```

Comment:
is this even valid syntax? as far as i'm aware there's no such key as "sprite"

Reply:
yes this is taken directly from the [Geode SDK Resource Page](https://docs.geode-sdk.org/mods/resources)